src/storybook/decorators/vertical-stories.jsx

```diff
@@ -14,3 +14,5 @@ export function VerticalStories(Story) {
     </div>
   );
 }
+
+export default VerticalStories;
```

Comment:
I think we do not want to export this... but not sure

Reply:
Removed from `published-storybook-components`